### PR TITLE
release: 3.6.7

### PR DESCRIPTION
**Pull request by AhyoungRyu**
### Feats:
* Added a new ImageGrid UI component (for internal use only) (#703)
* Introduced `fetchChannelList` to the `ChannelListContext`.
  * Implemented a custom hook function `useFetchChannelList`.
  * Utilized this function to fetch the channel list within the `ChannelListUI` component.
  * Added relevant tests for this function.
  * Provided the method through the `ChannelListContext`: `fetchChannelList`.
  Example Usage:
    ```jsx
    import SendbirdProvider from '@sendbird/uikit-react/SendbirdProvider'
    import useSendbirdStateContext from '@sendbird/uikit-react/useSendbirdStateContext'
    import { ChannelListProvider, useChannelListContext } from '@sendbird/uikit-react/ChannelList/context'

    const isAboutSame = (a, b, px) => (Math.abs(a - b) <= px);

    const CustomChannelList = () => {
      const {
        allChannels,
        fetchChannelList,
      } = useChannelListContext();

      return (
        <div
          className="custom-channel-list"
          onScroll={(e) => {
            const target = e.target;
            if (isAboutSame(target.clientHeight + target.scrollTop, target.scrollHeight, 10)) {
              fetchChannelList();
            }
          }}
        >
          {allChannels.map((channel) => {
            return // custom channel list item
          })}
        </div>
      );
    };

    const CustomApp = () => {
      return (
        <div className="custom-app">
          <SendbirdProvider ... >
            <ChannelListProvider ... >
              <CustomChannelList />
            </ChannelListProvider>
          </SendbirdProvider>
        </div>
      );
    };
    ```
### Fixes:
* Removed duplicated getEmoji API call from the `useGetChannel` hook (#705).
* Fixed missing `SEND_MESSAGE_FAILED` event publishing (#704):
  * Addressed the failure state in `sendbirdSelectors.getSendUserMessage` and published the `SEND_MESSAGE_FAILED` event.
  * Corrected typo `SEND_MESSAGEGE_FAILURE`.

### Chores:
* Added a troubleshooting guide to the README. (#702)
* Made minor improvements to the onboarding process. (#701)

### CODE REVIEW

**Comment by sendbird-sdk-deployment** [comment]
[Creating Ticket] 🔖 Done https://sendbird.atlassian.net/browse/SDKRLSD-944

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,68 @@
...
+* Fixed missing `SEND_MESSAGE_FAILED` event publishing (#704):
+  * Addressed the failure state in `sendbirdSelectors.getSendUserMessage` and published the `SEND_MESSAGE_FAILED` event.
+  * Corrected typo `SEND_MESSAGEGE_FAILURE`.
+
+* Resolved an issue where the `allChannels` couldn't be updated due to state updates occurring only through an internal dispatcher (#708):
+  * Introduced `fetchChannelList` to the `ChannelListContext`.
+  * Implemented a custom hook function `useFetchChannelList`.
+  * Utilized this function to fetch the channel list within the `ChannelListUI` component.
+  * Added relevant tests for this function.
+  * Provided the method through the `ChannelListContext`: `fetchChannelList`.
```
Comment (sravan-s) [inline CHANGELOG.md:18]: feature ~

Comment (AhyoungRyu) [reply CHANGELOG.md:10]: https://github.com/sendbird/sendbird-uikit-react/pull/706/commits/624d75fb13639e6683fd5da4219521bf63f952c2

**Review by sravan-s — APPROVED** [review]
LGTM

**Pull request merged**